examples/onnxrt/image_recognition/onnx_model_zoo/resnet50/quantization/ptq_static/requirements.txt

```diff
@@ -4,3 +4,4 @@ torch
 torchvision
 onnxruntime-extensions; python_version < '3.11'
 pillow>=8.2.0 # not directly required, pinned by Snyk to avoid a vulnerability
+transformers
```

Comment:
should it be added only if onnxruntime-extensions >= 1.14.0 ?

Reply:
Conditional judgment is not supported. Do you mean to specify onnxruntime-extensions>= 1.14.0?

Reply:
I mean onnxruntime-extensions is only installed with python_version < 3.11, so transformers should be installed when onnxruntime-extensions is installed and onnxruntime-extensions >= 1.14.0